chrome_manifest_v3/views/class_popupviewforms.js

```
@@ -0,0 +1,963 @@
+        initials = me.username.slice(0, 1);
+      }
+
+      avatarText = initials.toUpperCase();
+      $('#g2tAvatarImgOrText', this.parent.$popup).text(avatarText);
+    } else {
+      $('#g2tAvatarImgOrText', this.parent.$popup).html(
+        '<img width="30" height="30" alt="' +
+          me.username +
+          '" src="' +
+          avatarSrc +
+          '">'
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)